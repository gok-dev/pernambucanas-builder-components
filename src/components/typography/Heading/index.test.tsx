// import { renderWithTheme } from 'utils/helpers'
// import { Heading } from './index'

// describe('<Heading />', () => {
//   it('shold render the heading', () => {
//     const { container } = renderWithTheme(<Heading />)
//     expect(container.firstChild).toMatchSnapshot()
//   })
// })
