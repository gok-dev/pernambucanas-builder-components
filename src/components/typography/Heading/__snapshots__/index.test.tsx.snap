// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> shold render the heading 1`] = `
.c0 {
  font-size: 34px;
}

<h1
  class="c0"
/>
`;
